src/sagas/index.js

```diff
@@ -381,7 +382,7 @@ export function* stopProgress ({ vmId, poolId, name, result }) {
     yield getSingleInstance({ vmId, poolId })
   }
 
-  yield put(actionInProgress(Object.assign(vmId ? { vmId } : { poolId }, { name, started: false })))
+  yield put(actionInProgress(Object.assign(useVmId ? { vmId } : { poolId }, { name, started: false })))
```

Comment:
If `vmId` was retrieved from `result` then I think we should not fire `poolActionInProgress`.
I'd rather fire `vmActionInProgress` with the new ID i.e. (pseudo-code)
```
if (result && ..) {
    vmId = vmId || result.vm.id
    ...
}

if (vmId)  {
   vmActionInProgress({vmId},...)
} else {
   poolActionInProgress({poolId},...)
}
```

Reply:
It make sense, @rszwajko  I changed it.
Thanks :)